router.js

```diff
@@ -1,14 +1,11 @@
 import express from 'express';
-import {AsyncLocalStorage, executionAsyncId, executionAsyncResource} from 'async_hooks';
+import {storage} from './storage.js'
 
 let router = new express.Router();
 
-
 router.get('/', async (req, res) => {
-    let storage = new AsyncLocalStorage();
-    let store = storage.getStore();
+    const store = storage.getStore();
```

Comment:
you don't re-create a new instance for each call

Reply:
here, i got it, but for example what about executionAsyncResource() -> as this provide you your current state everywhere, without make some singleton storage file, as in example, i can get directly by symbol store in this execution context, Maybe AsyncLocalStorage should have static method which always will return current kResourceStore  if exist???

Reply:
There are discussions on wether `executionAsyncResource` is the right thing to use internally.
Actually, there might be some heavy changes in how the Async Hooks module work. However, `AsyncLocalStorage` is planned to be stabilized soon and other changes will not impact it.

I can't recommend doing anything with `executionAsyncResource` as it might change at any in the future. Also, the fact that `AsyncLocalStorage` relies on it is something that might change too (as long as something is undocumented, it is suject to change)

Reply:
okay, so for now better use some singleton file for ALS